sdk/tables/Azure.Data.Tables/samples/README.md

```diff
@@ -0,0 +1,28 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-tables
+name: Azure Tables samples for .NET
+description: Samples for the Azure.Data.Tables client library
+---
+
+# Azure Tables client SDK Samples
+Description of Azure Tables. Covers following functions: 
+* Create and delete a table
```

Comment:
In general this looks great. I will suggest to remove the functions we are currently not implementing so that if someone comes to this page, they understand what information currently they can find